server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/mapper/TagBase.java

```diff
@@ -0,0 +1,109 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.mapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.opentosca.toscana.core.util.Preferences;
+import org.opentosca.toscana.plugins.kubernetes.docker.mapper.model.DockerImage;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
I would consider this a Repository, because its used to persist data. maybe replace `@Component` with `@Repository` but thats just cosmetics.

Reply:
If it does not cause any issues you could also consider excluding this from the regular tests. Using:
`@Profile("!" + Profiles.EXCLUDE_BASE_IMAGE_MAPPER)`

The BaseImageMapper class does that to prevent loading every time a context is created. This won't change with this update, because the downloaded file is deleted after every test.